lib/logstash/outputs/opensearch/http_client/pool.rb

```diff
@@ -232,7 +237,11 @@ def healthcheck!
           # If no exception was raised it must have succeeded!
           logger.warn("Restored connection to OpenSearch instance", url: url.sanitized.to_s)
           # We reconnected to this node, check its version
-          version = get_version(url)
+          if skip_healthcheck == false
+            version = get_version(url)
+          else
+            version = "#{default_server_major_version}.0.0"
```

Comment:
We're defining default version as 2, should this also be kept as 2 instead of `2.0.0` for consistency?

Reply:
The parsing function used to pick out the major version expects the version in this format. Making this 2 was resulting in additional if checks and code clutter.